Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -80,12 +80,12 @@ echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometh
 
 Following is an example to configure the Prometheus endpoint in the CephCluster CR.
 
-    ```YAML
+```YAML
```

Comment:
Interesting that later in the doc there are similar instances, but they are rendered properly, perhaps because they under under bulleted points.